src/mem3/src/mem3_rpc.erl

```diff
@@ -41,6 +43,11 @@
 -include("mem3.hrl").
 -include_lib("couch/include/couch_db.hrl").
 
+% "Pull" is a bit of a misnomer here, as what we're actually doing is
+% issuing an RPC request and telling the remote node to push updates to
+% us. This lets us reuse all of the battle-tested machinery of mem3_rpc.
+pull_from_seed(Seed) ->
```

Comment:
This seems useful in general and maybe rename this to pull_replication so it matches with the pull_replication_rpc local callback?

Reply:
Good idea will do